<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodificadorDlg</class>
 <widget class="QDialog" name="CodificadorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codificador</string>
  </property>
  <widget class="QRadioButton" name="rb4Bandas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>4 Bandas</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb5Bandas">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>5 Bandas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCodificar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Codificar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValorRes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Banda1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Banda2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Banda3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Banda4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="UnidadBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ohms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>k Ohms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M Ohms</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G Ohms</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
